include/dlr.h

```diff
@@ -47,6 +47,17 @@ typedef void (*DLRFreeFunctionPtr)(void*);
 typedef void* (*DLRMemalignFunctionPtr)(size_t, size_t);
 #endif
 
+#ifndef DLR_MODEL_ELEM
+#define DLR_MODEL_ELEM
+enum DLRModelElemType { HEXAGON_LIB, NEO_METADATA, TVM_GRAPH, TVM_LIB, TVM_PARAMS, RELAY_EXEC };
+typedef struct ModelElem {
+  const DLRModelElemType type;
```

Comment:
can we add constructors for the struct to simplify setting just a path, or data & size?

Reply:
Looks like in C they just use `{a,b,c}` to create and init a struct.